### DISCUSSION TITLE
[Bug]: pimcore.sock not created on /var/run/php/pimcore.sock by installation

### DISCUSSION THREAD

**Discussion by leitge** [pimcore/pimcore#14556] [Q&A / Support]
### Expected behavior

During Pimcore installation, a pimcore.sock should be created. 

### Actual behavior

When installing Pimcore, no pimcore.sock is created in /var/run/php/pimcore.sock. Installation went through successfully.

Even with repeated attempt and previous creation of the directory /var/run/php/ no change. 

Without Pimcore.sock unfortunately no operation by NGINX is possible. 

System:
AWS EC2
Apache is not installed.
PHP8.1
Pimcore 10.5.18

Info:
Also with Bitnami Standard Pimcore package no pimcore.sock is created.

### Steps to reproduce

Take Bitnami Default Package or install Pimcore in current release in default ec2 instance

**Comment by brusch** [comment]
Thanks for reporting the issue, however we're not the maintainer of the Bitnami images, so there's actually nothing we can do here.

**Comment by leitge** [comment]
Hey brusch
Thanks for the quick response. The reference to the Bitnami package was only to show a second reproduction. The problem exists independently with any installation. The actual project is **not** based on the Bitnami package.

Also, there are already others here with the same problem.

**Comment by leitge** [comment]
#13740 "pimcore.sock file is not created"

@brusch  here you have same discussion linked from end 2022. 
Is the pimcore.sock installation specific or an generic file? any way to get it and save it manually for the moment, as an workaround?
